csharp/Platform.Collections/Arrays/ArrayString.cs

```diff
@@ -1,18 +1,44 @@
 ﻿using System.Runtime.CompilerServices;
 using Platform.Collections.Segments;
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-
 namespace Platform.Collections.Arrays
 {
     public class ArrayString<T> : Segment<T>
     {
+        /// <summary>
+        /// <para>Initializes an object of the Segment class with the passed data.</para>
+        /// <para>Инициализирует объект класса Segment с переданными данными.</para>
```

Comment:
Look here https://github.com/linksplatform/Ranges/blob/master/csharp/Platform.Ranges/Range%5BT%5D.cs#L33-L58 as an example.
You can also use references to class here: <see cref="ArrayString{T}"/>.
But you should use reference to ArrayString and not Segment.
These constructors create instances of ArrayString class and not Segment.

Because you this class inherits from Segment class, I suggest to cover https://github.com/linksplatform/Collections/blob/master/csharp/Platform.Collections/Segments/Segment.cs with comments first.